docs/contributing-analyzer.md

```diff
@@ -12,14 +12,15 @@ In general, it is best to run commands from the Visual Studio Developer Command
 
 ## Developing with Visual Studio 2017
 
-1. [Visual Studio 2017 Version 15.7](https://www.visualstudio.com/vs/preview/)
+1. [Visual Studio 2017 Version 15.9](https://my.visualstudio.com/Downloads?q=visual%20studio%202017&wt.mc_id=o~msft~vscom~older-downloads)
```

Comment:
The link did not work for me for first try. It shows this error message: Sorry, we couldn't find any downloads for you. Returning to home and accepting some terms and conditions made it work. I guess that's fine.

I suggest to shorten the link like this just to keep it simple:
https://my.visualstudio.com/Downloads?q=Visual%20Studio%202017